pkg/sql/parser/eval.go

```diff
@@ -1846,6 +1846,8 @@ type EvalPlanner interface {
 	// QualifyWithDatabase resolves a possibly unqualified table name into a
 	// normalized table name that is qualified by database.
 	QualifyWithDatabase(ctx context.Context, t *NormalizableTableName) (*TableName, error)
+
+	IncrementSequence(context context.Context, seqName string) (int64, error)
```

Comment:
This is a very specific method to have in this interface. Since this is the first function which makes KV writes, not sure what the best way is to plumb things through.